xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/delete.vm

```diff
@@ -238,16 +238,36 @@
 #######################################################
 #macro(displayConfirmMessage)
   #if($xwiki.hasRecycleBin())
-    #if($hasInlinks)
-      <p><strong>$services.localization.render('core.recyclebin.confirmWithInlinks')</strong></p>
+    #if($services.refactoring.isAllowedToSkipTheRecycleBin())
+      <div class="xform">
+        <dl>
+          <dt>
+            $escapetool.xml($services.localization.render('core.recyclebin.wantToSkip'))
+          </dt>
+          <dd>
+            <label class="xwiki-form-listclass">
```

Comment:
I might be wrong here but AFAIR a label element should always have a "for" attribute that points to the input element the label is for, for accessibility reason. Example there: https://www.xwiki.org/xwiki/bin/view/Documentation/DevGuide/FrontendResources/VerticalForms/#H1.Labels. You can check easily if I'm right or not by executing the webstandards test, we should check this.

Reply:
From https://www.w3schools.com/tags/tag_label.asp
> The for attribute of <label> must be equal to the id attribute of the related element to bind them together. A label can also be bound to an element by placing the element inside the <label> element. 

Here the label is a the parent tag of the radio button, consequently clicking on the label text triggers the selection of the corresponding radio button.

Reply:
OK, just try to run the webstandards test locally to be sure: would be actually useful to also check other possible problems.

Reply:
OK thanks.